From f6d76981b1b31a389c30239e0b10332ddacceccf Mon Sep 17 00:00:00 2001
From: Jiri Slaby <jslaby@suse.cz>
Date: Mon, 4 Nov 2019 09:56:20 +0100
Subject: [PATCH] CMakeLists: support different lib dirs

On 64bit systems, the usual destination for libraries is /usr/lib64,
not /usr/lib. So add CMAKE_INSTALL_LIBDIR and allow overriding the
the default.
---
 CMakeLists.txt | 6 ++++--
 1 file changed, 4 insertions(+), 2 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 465fd913..10c8bf08 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -220,8 +220,8 @@ set_target_properties(minisat_simp       PROPERTIES OUTPUT_NAME "minisat")
 install(TARGETS minisat minisat_core minisat_simp
         EXPORT ${MINISAT_EXPORT_NAME}
         RUNTIME DESTINATION bin
-        LIBRARY DESTINATION lib
-        ARCHIVE DESTINATION lib)
+        LIBRARY DESTINATION ${CMAKE_INSTALL_LIBDIR}
+        ARCHIVE DESTINATION ${CMAKE_INSTALL_LIBDIR})
 
 install(DIRECTORY minisat/mtl minisat/utils minisat/core minisat/simp
         DESTINATION include/minisat
-- 
2.26.2

